app/forms/state_file/md_tax_refund_form.rb

```diff
@@ -24,9 +37,20 @@ class MdTaxRefundForm < QuestionsForm
       validates :routing_number, presence: true, confirmation: true, routing_number: true
       validates :routing_number_confirmation, presence: true
 
+      validates :bank_authorization_confirmed, acceptance: { accept: 'yes', message: ->(_object, _data) { I18n.t("state_file.questions.md_tax_refund.md_bank_details.bank_authorization_confirmation_error") }}
```

Comment:
i don't see anything in the form spec about this field

Reply:
addressed 32a3033da321cde88ea8177db00c66950639f251